### PR TITLE
Update convert_qcow2_ova.py

### PR DESCRIPTION
**Pull request by sudeeshjohn**
Fixing the distribution check

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/8#" title="Author self-approved">sudeeshjohn</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: scripts/images/convert_qcow2_ova.py
Comment (mkumatag) [inline scripts/images/convert_qcow2_ova.py:87]: I insist to include these just after the bash interpreter to catch any failures!

```
set -o errexit
set -o nounset
set -o pipefail
set -o xtrace
```

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Review by mkumatag — APPROVED** [review]

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/8#pullrequestreview-490738610" title="Approved">mkumatag</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/8#" title="Author self-approved">sudeeshjohn</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
